portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -87,19 +85,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
   @Override
   public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String tokenId = request.getParameter("id");
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
why move this up here?  Declare it as close to where it's used as possible unless there is some startup latency.